src/handle_wrap.cc

```diff
@@ -37,6 +38,17 @@ void HandleWrap::Unref(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void HandleWrap::IsRefed(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  HandleWrap* wrap = Unwrap<HandleWrap>(args.Holder());
+
+  if (IsAlive(wrap)) {
+    bool refed = (wrap->flags_ & kUnref) == 0;
+    args.GetReturnValue().Set(Boolean::New(env->isolate(), refed));
+  }
+}
```

Comment:
returning nothing == returning undefined. but the JS signature returns bool. The question this is trying to answer is "Can this handle hold open the event loop?" So if `!IsAlive()` means should return false.